frappe/public/js/frappe/ui/page.js

```diff
@@ -397,6 +397,49 @@ frappe.ui.Page = Class.extend({
 		this.current_view_name = name;
 
 		this.views[name].toggle(true);
+	},
+	add_document_flow: function(frm){
+		var me = this;
+		$('.module-flow-section').empty();
+		if (frm.doctype) {
+			var module = frappe.get_meta(frm.doctype).module
+			console.log(module)
+			var module_flow = frappe.boot.module_flow[module][frm.doctype];
```

Comment:
why boot? boot is only for data that is loaded on session start

if you want global, use `frappe.module_flow`